index.html

```diff
@@ -56,5 +56,182 @@ <h3>LET’S CONNECT</h3>
         </ul>
       </div>
     </section>
+
+    <section class="works">
+      <div class="card1">
+        <img src="images\Snapshoot Portfolio.png" alt="Page example display" />
+        <h2>Tonic</h2>
+        <ul class="date">
+          <li class="canopy">CANOPY</li>
+          <li class="year">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p class="selection-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-buttons">
+          <li>
+            <a href="#"><img src="images\html.png" alt="html button" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images\css.png" alt="css button" /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="images\javascript.png" alt="javascript button"
+            /></a>
+          </li>
+        </ul>
+        <button class="project-button">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="images\avail.png" alt="Page example display" />
+        <h2>Multy-Post Stories</h2>
```

Comment:
@Nyame-Wolf Hi, can you please clarify the problem with the margins I'm using? Because I'm using the same padding value from the Figma template (16px) but less margin left and right (14px instead of 24px on the template).
Also, do I need to change it to a fixed width like in the template?